Reaqtor/Samples/Remoting/Reaqtor.Remoting.Deployable/CoreDeployable.cs

```diff
@@ -150,8 +159,10 @@ protected virtual async Task DefineOperators(ReactiveClientContext context)
             await context.DefineObservableAsync<IAsyncReactiveQbservable<T>, bool>(new Uri(Client.Constants.Observable.Any.NoArgument), source => source.AsSync().Any().AsAsync(), null, CancellationToken.None).ConfigureAwait(false);
             await context.DefineObservableAsync<IAsyncReactiveQbservable<T>, Expression<Func<T, bool>>, bool>(new Uri(Client.Constants.Observable.Any.Predicate), (source, predicate) => source.AsSync().Any(predicate).AsAsync(), null, CancellationToken.None).ConfigureAwait(false);
             await context.DefineObservableAsync<IAsyncReactiveQbservable<T1>, IAsyncReactiveQbservable<T2>, Expression<Func<T1, T2, R>>, R>(new Uri(Client.Constants.Observable.CombineLatest.ObservableFunc), (source, otherSource, selector) => source.AsSync().CombineLatest(otherSource.AsSync(), selector).AsAsync(), null, CancellationToken.None).ConfigureAwait(false);
+            await context.DefineObservableAsync<IAsyncReactiveQbservable<T>, T, bool>(new Uri(Client.Constants.Observable.Contains.Element), (source, element) => source.AsSync().Contains(element).AsAsync(), null, CancellationToken.None).ConfigureAwait(false);
```

Comment:
Subtle but important, this should call the `Contains` overload with an `IEqualityComparer<T>` by supplying `DataTypeEqualityComparer<T>.Default`.

In the context of Reaqtor, we enforce the use of data model types which have value-based equality. For example, see how we define `Distinct` here:

https://github.com/reaqtive/reaqtor/blob/main/Reaqtor/Samples/Remoting/Reaqtor.Remoting.Deployable/CoreDeployable.cs#L158

Note that we don't provide access to overloads with a custom `IEqualityComparer<T>` because there's currently no quotation type that users can leverage client-side to create a quoted expression-tree-based representation of a comparer (e.g. referring to a known comparer using a `Uri`-based identifier, or using a creation operator that takes an `Expression<Func<T, T, bool>>` for equality comparison).